src/Router.js

```diff
@@ -17,20 +17,20 @@ import MainMinyoungKang from "./pages/minyoungKang/Main/Main";
 import LoginMinyoungKang from "./pages/minyoungKang/Login/Login";
 
 const Router = () => {
-	return (
-	<BrowserRouter>
-        <Routes>
-            <Route path="/hyoeunjung-main" element={<MainHyoeunJung />} />
-            <Route path="/hyoeunjung-login" element={<LoginHyoeunJung />} />
-            <Route path="/hyungminjeon-main" element={<MainHyungminJeon />} />
-            <Route path="/hyungminjeon-login" element={<LoginHyungminJeon />} />
-            <Route path="/jinheekim-main" element={<MainJinheeKim />} />
-            <Route path="/jinheekim-login" element={<LoginJinheeKim />} />
-            <Route path="/minyoungkang-main" element={<MainMinyoungKang />} />
-            <Route path="/minyoungkang-login" element={<LoginMinyoungKang />} />
-        </Routes>
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/hyoeunjung-main" element={<MainHyoeunJung />} />
+        <Route path="/hyoeunjung-login" element={<LoginHyoeunJung />} />
+        <Route path="/hyungminjeon-main" element={<MainHyungminJeon />} />
+        <Route path="/hyungminjeon-login" element={<LoginHyungminJeon />} />
```

Comment:
```suggestion
        <Route path="/hyungminjeon-main" element={<MainHyungminJeon />} />
        <Route path="/" element={<LoginHyungminJeon />} />
```
형민님의 로그인페이지를 root경로로 지정해서 좀 더 편하게 개발 진행하셔도 괜찮습니다!